content/terraform/v1.13.x/docs/cli/commands/import.mdx

```diff
@@ -5,6 +5,12 @@ description: The `terraform import` command imports existing resources into Terr
 
 # `terraform import` command reference
 
+<Note>
+
+The beta release of Terraform 1.14 lets you configure queries to search your existing infrastructure for resources and generate configuration to import them. Refer to the [Import existing resources in bulk](/terraform/language/v1.14.x/import/bulk) beta documentation for more information.
```

Comment:
```suggestion
Terraform `v1.14.x (beta)` lets you search existing infrastructure for resources, generate configuration, and import them in bulk. Refer to the [Import existing resources in bulk](/terraform/language/v1.14.x/import/bulk) beta documentation for more information.
```
Trying for a shorter paragraph. 
Again, I could go either way on formatting the version because the backticks deviate from the style guide. 
Same recommendation about placing this after the intro paragraph.